### ISSUE TITLE
Layout/MultilineMethodArgumentLineBreaks False Positive On Multiline Assignment

### ISSUE THREAD

**Issue by jfelchner** [rubocop-hq/rubocop#6936]
This is a pretty major bug.  The `Layout/MultilineMethodArgumentLineBreaks` cop is throwing errors when an assignment spans multiple lines.  I'm assuming because it sees the assignment operator as a "method" (which it technically is but should not be treated like a method call for the purposes of this cop).

Original PR: #6826 

/cc @maxh 

--------

## Expected behavior

The following code should not throw an error.

```
class Thing
  def call
    bar['foo'] = ::Time.zone.at(
                   huh['foo'],
                 )
  end
end
```

## Actual behavior

It throws an error.

## Steps to reproduce the problem

See code snippet above.

## RuboCop version

```
0.67.2 (using Parser 2.6.2.1, running on ruby 2.6.2 x86_64-darwin17)
```

**Comment by maxh** [comment]
Apologies for delay; didn't get an email for this. Fix: https://github.com/rubocop-hq/rubocop/pull/7072
